### PR TITLE
Hand signals enhancement 

### PR DESCRIPTION
**Pull request by jokoho48**
Port THS to ACE3 after #865

### CODE REVIEW

**Comment by jonpas** [comment]
My thoughts are the same as in the old PR https://github.com/acemod/ACE3/pull/2107#issuecomment-129654944 about this.

File: addons/interaction/keys.sqf [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        "ACE3 Gestures",
+        _currentName,
+        format ["%1 %2", localize format[LSTRING(%1), _currentName],localize LSTRING(FREELOOK)],
+        _code,
+        {},
+        [_Key,  if (_Key == 0) then {[false, false, false]} else { [false, true, false] }],
+        false
+    ] call cba_fnc_addKeybind;
+
+    if (56 in (actionKeys "LookAround")) then {
```
Comment (commy2) [inline addons/interaction/keys.sqf]: ?
This is weird as fuck. There is probably a better way for this.

File: addons/interaction/functions/fnc_playSignal.sqf [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+_gesture = if (_this select [0,2] == "BI") then {
+    _this select [2]
+} else {
+    if (((animationState ACE_player) select [0, 12]) in ["amovpercmstp", "amovpercmwlk", "amovpercmtac"] && weaponLowered ACE_player) then {
+        format ["%1StandLowered", _this]
+    } else {
+        _this
+    };
+};
+player playAction _gesture;
```
Comment (commy2) [inline addons/interaction/functions/fnc_playSignal.sqf]: `player`

File: addons/interaction/keys.sqf (hunk 1 above)
Comment (gienkov) [inline addons/interaction/keys.sqf]: It's so you can use gestures keybinds while freelook (holding ALT). Works ok for me here.

Comment (jokoho48) [inline addons/interaction/keys.sqf]: yeah i ported this over from THS

Comment (commy2) [inline addons/interaction/keys.sqf]: But what if the key no longer is alt/56 ?
I'm strongly against stuff that only works with default keybinds.

File: addons/gestures/CfgVehicles.hpp [hunk 3]
```diff
@@ -0,0 +1,158 @@
+class CfgVehicles {
+
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class ACE_Gestures {
```
Comment (jonpas) [inline addons/gestures/CfgVehicles.hpp]: `GVAR`? And for all sub-actions as well.

File: addons/gestures/stringtable.xml [hunk 4]
```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Gestures">
+        <Key ID="STR_ACE_Gestures_NAME">
```
Comment (jonpas) [inline addons/gestures/stringtable.xml]: Lowercase `NAME`.

File: addons/gestures/stringtable.xml [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+        <!-- WARNING -->
+        <Key ID="STR_ACE_Gestures_warning">
+            <English>Warning</English>
+            <French>Attention</French>
+            <German>Achtung</German>
+            <Polish>Uwaga</Polish>
+        </Key>
+
+        <!-- FREE LOOK -->
+        <Key ID="STR_ACE_Gestures_FREELOOK">
```
Comment (jonpas) [inline addons/gestures/stringtable.xml]: Lowercase `FREELOOK`.

File: addons/gestures/stringtable.xml [hunk 6]
```diff
@@ -0,0 +1,191 @@
...
+            <Czech>Stát</Czech>
+            <French>Halte</French>
+            <Russian>Замереть</Russian>
+            <Hungarian>Állj</Hungarian>
+            <Portuguese>Alto</Portuguese>
+            <Italian>Fermi</Italian>
+        </Key>
+
+        <!-- STOP -->
+        <Key ID="STR_ACE_Gestures_stop">
```
Comment (jonpas) [inline addons/gestures/stringtable.xml]: Capitalize gesture name, all from here down.

**Comment by Glowbal** [comment]
@jokoho48 what is the status on this?

**Comment by jokoho48** [comment]
i currently cant test because my test Enviroment stuck and i cant build/test it

**Comment by PabstMirror** [comment]
Seems to be working good with keybinds or interaction menu.

**Comment by gienkov** [comment]
Tested on dedicated, works.

**Comment by WadeDP** [comment]
The unbinded gestures don't work when they are binded to something, tested with only CBA and ACE. Also would recommend to have a setting to remove gestures action from self-action menu.

**Comment by jonpas** [comment]
@WadeDP please open a new issue.

**Pull request merged**